ibm_catalog.json

```diff
@@ -183,14 +237,14 @@
                   ]
                 },
                 {
-                  "key": "watsonx_ai_studio_instance_name"
+                  "key": "existing_watsonx_ai_runtime_instance_crn"
                 },
                 {
                   "key": "watsonx_ai_runtime_instance_name"
                 },
                 {
                     "key": "watsonx_ai_runtime_plan",
-                    "default_value": "v2-professional",
+                    "default_value": "v2-standard",
```

Comment:
default values should not be added to ibm_catalog.json so that they are pulled directly from the terraform code. Otherwise we need to maintain in 2 places.